packages/kit/types/index.d.ts

```diff
@@ -113,7 +114,7 @@ export interface Config {
 		exports?: (filepath: string) => boolean;
 		files?: (filepath: string) => boolean;
 	};
-	preprocess?: any;
+	preprocess?: SvelteOptions['preprocess'];
```

Comment:
reverting this, it's preventing the docs from deploying and is no longer necessary for this PR

```suggestion
	preprocess?: any;
```